src/pages/core/plugins/freeze-delegate.md

```
@@ -114,14 +133,74 @@ await approvePluginAuthority(umi, {
+    .collection(Some(ctx.accounts.collection))
+    .authority(Some(ctx.accounts.authority))
+    .payer(ctx.accounts.payer)
+    .plugin_type(PluginType::FreezeDelegate)
+    .new_authority(PluginAuthority::Address { address: ctx.accounts.new_authority.key() })
+    .invoke()?;
+```
+
+{% /dialect %}
 {% dialect title="Rust" id="rust" %}
-_coming soon_
+```rust
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)